### ISSUE TITLE
Ember Sinon asset in production build

### ISSUE THREAD

**Issue by blimmer** [csantero/ember-sinon#21]
I'm seeing this in my production builds:

```js
define('ember-cli-example-app-for-github/acceptance-tests/sinon', ['exports', 'ember-sinon/acceptance-tests/sinon'], function (exports, sinon) {

	'use strict';



	exports['default'] = sinon['default'];

});
```

Seems weird that an acceptance-test module is in the prod build.

**Comment by blimmer** [comment]
Here's a repo to reproduce the issue: https://github.com/blimmer/ember-cli-example-app-for-github/commit/917e58708797a2809a85d0c9d3e49a48240f961b

**Comment by csantero** [comment]
Fixed and released in 0.2.2. Thanks for spotting this.
